packages/appsync-modelgen-plugin/src/visitors/appsync-dart-visitor.ts

```diff
@@ -610,13 +611,18 @@ export class AppSyncModelDartVisitor<
   }
 
   protected generateSchemaField(model: CodeGenModel, declarationBlock: DartDeclarationBlock): void {
+    const pluralFields = this.config.improvePluralization ? `modelSchemaDefinition.listPluralName = "${
```

Comment:
As mentioned in the comment, Flutter need option 2 for the change (having all three fields instead of replacing)

Reply:
**Correction:** amplify-flutter should take the **option 1**, the dart-visitor should not make any changes.

Reply:
@AaronZyLee removed